components/dashboard/src/usage/download/DownloadUsage.tsx

```diff
@@ -129,7 +129,7 @@ const DownloadUsageToast: FC<DownloadUsageToastProps> = ({ attributionId, endDat
         <div className="flex flex-row items-start justify-between space-x-2">
             <div>
                 <span>Usage export complete.</span>
-                <p>
+                <p className="dark:text-gray-500">
```

Comment:
**thought**: I'm wondering if there's a better place for this, as the second line is using the correct color.